X16 Reference - 09 - VERA Programmer's Reference.md

```diff
@@ -455,11 +462,28 @@ The VERA consists of:
 | $1FA00 - $1FBFF | Palette                    |
 | $1FC00 - $1FFFF | Sprite attributes          |
 
+The X16 KERNAL uses the following video memory layout:
+
+| Addresses     | Description                                               |
+|---------------|-----------------------------------------------------------|
+| $00000-$12BFF | 320x240@256c Bitmap                                       |
+| $12C00-$12FFF | *unused* (1024 bytes)                                     |
+| $13000-$1AFFF | Sprite Image Data (up to $1000 per sprite at 64x64 8-bit) |
+| $1B000-$1EBFF | Text Mode                                                 |
+| $1EC00-$1EFFF | *unused* (1024 bytes)                                     |
+| $1F000-$1F7FF | Charset                                                   |
+| $1F800-$1F9BF | *unused* (1024 bytes)                                     |
+| $1F9C0-$1F9FF | VERA PSG Registers (16 x 4 bytes)                         |
+| $1FA00-$1FBFF | VERA Color Palette (256 x 2 bytes)                        |
+| $1FC00-$1FFFF | VERA Sprite Attributes (128 x 8 bytes)                    |
+
+Application software is free to use any part of video RAM if it does not use the
+corresponding KERNAL functionality. To restore text mode, call `CINT` ($FF81).
+
```

Comment:
There was historically a reason to keep VERA and X16-specific things separate.  Is there a way we can phrase this so that it's abundantly clear that the VERA has no affinity to this particular layout (up to and not including $1F9C0), and that it only matters for the X16's kernal and ROM routines?

It would be nice to clearly convey the understanding to game/app programmers that they can use any memory layout they wish using the layer config and the sprite registers.